scripts/hail_batch/hgdp1kg_tobwgs_nfe_pc_relate/hgdp_1kg_tob_wgs_nfe_pc_relate.py

```diff
@@ -0,0 +1,57 @@
+"""
+Estimate kinship for nfe samples from the HGDP/1KG +
+tob-wgs datasets.
+
+"""
+
+import hail as hl
+import pandas as pd
+from analysis_runner import bucket_path, output_path
+
+
+HGDP1KG_TOBWGS = bucket_path(
+    '1kg_hgdp_densified_pca_new_variants/v0/hgdp1kg_tobwgs_joined_all_samples.mt'
+)
+
+
+def query():
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    mt = hl.read_matrix_table(HGDP1KG_TOBWGS)
+    # Get samples from the specified population only
+    mt = mt.filter_cols(
+        (mt.hgdp_1kg_metadata.population_inference.pop == 'nfe')
+        | (mt.s.contains('TOB'))
+    )
+
+    # Perform kinship test with pc_relate
+    pc_rel = hl.pc_relate(mt.GT, 0.01, k=20, statistics='kin')
```

Comment:
If we're filtering down to outliers later, do we really need to compute this for the whole table?

Reply:
I wanted to do the analysis exactly the same. I think the `pc_relate` function uses information from the entire population: https://hail.is/docs/0.2/methods/relatedness.html#hail.methods.pc_relate

Reply:
Fair enough! However, do you want to save `pc_rel` in that case, so we don't have to recompute that in case you want to inspect other elements?